src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -40,6 +41,15 @@ import {
 
 import reducer, { init } from './reducer';
 
+const USERS_AUTO_SUGGEST_URL = `${configuration.USERS_ENDPOINT}?is_active=true&username=`;
```

Comment:
The `UserAutoSuggest` is like `PDOKAutosuggest` an specific implementation of the `AutoSuggest` control. Shell we extract this to a separate component? This would keep the FilterForm component a little bit cleander.

Reply:
That will only extract these lines of code: https://github.com/Amsterdam/signals-frontend/pull/1310/files#diff-985e301408102dce7b18001ad6c36fedab4917573589c06456ba167ce9ee3a37R44-R52
It is not being used anywhere else (yet), so I'm not sure if it's worth extracting.